python/seldon_core/utils.py

```diff
@@ -10,6 +10,14 @@
 from google.protobuf.json_format import MessageToDict, ParseDict
 from google.protobuf.struct_pb2 import ListValue
 
+from seldon_core.env_utils import (
```

Comment:
we could just  (if we put this into `utils`)
```
from seldon_core import utils
```
and then use `utils.get_image_name` wherever needed

Though both works and and usually over-import so does not require a change now.